packages/applications/ssr/src/app/laureats/[identifiant]/garanties-financieres/page.tsx

```diff
@@ -49,22 +49,16 @@ export default async function Page({ params: { identifiant } }: IdentifiantParam
         return <ProjetNonSoumisAuxGarantiesFinancièresPage projet={projet} />;
       }
 
-      /**
-       * @todo à refacto suite à l'introduction d'une réponse de la query en utilisant @potentiel-libraries/monads
-       */
-
-      const garantiesFinancières = await tryToGetResource(
-        async () =>
-          await mediator.send<GarantiesFinancières.ConsulterGarantiesFinancièresQuery>({
-            type: 'Lauréat.GarantiesFinancières.Query.ConsulterGarantiesFinancières',
-            data: { identifiantProjetValue: identifiantProjet },
-          }),
-      );
+      const garantiesFinancières =
+        await mediator.send<GarantiesFinancières.ConsulterGarantiesFinancièresQuery>({
+          type: 'Lauréat.GarantiesFinancières.Query.ConsulterGarantiesFinancières',
+          data: { identifiantProjetValue: identifiantProjet },
+        });
 
       const props = mapToProps({
         projet,
         utilisateur,
-        garantiesFinancières,
+        ...(Option.isSome(garantiesFinancières) && { garantiesFinancières }),
```

Comment:
Je crois qu'on s'était dit qu'on évitait les spread au max, du coup voici un proposition : 

```suggestion
        garantiesFinancières : Option.isSome(garantiesFinancières) ? garantiesFinancières : undefined
```

Reply:
Ah bah j'ai fait la même remarque, je suis d'accord, bien plus lisible 👍